supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkProcess.scala

```diff
@@ -0,0 +1,93 @@
+package cromwell.backend.impl.spark
+
+import java.nio.file.Path
+
+import com.typesafe.scalalogging.StrictLogging
+import cromwell.core.{TailedWriter, UntailedWriter}
+import cromwell.core.PathFactory.EnhancedPath
+
+import scala.sys.process._
+import better.files._
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
+
+object SparkCommands {
+  val SPARK_SUBMIT = "spark-submit"
```

Comment:
Conform to standard naming conventions.

Same comment as above. If these are used in a single place, de-constantify them. This isn't enterprise java :)